resources/application-connector/values.yaml

```diff
@@ -23,37 +23,37 @@ global:
   containerRegistry:
     path: eu.gcr.io/kyma-project
   application_operator:
-    version: "PR-9406"
+    version: "2407bbc2"
   application_operator_tests:
-    version: "PR-9394"
+    version: "4eeef0b7"
   connector_service:
-    version: "PR-9406"
+    version: "2407bbc2"
   connector_service_tests:
-    version: "PR-9214"
+    version: "2ab8b878"
   connection_token_handler:
-    version: "PR-9406"
+    version: "2407bbc2"
   connection_token_handler_tests:
-    version: "PR-9406"
+    version: "2407bbc2"
   event_service:
     version: "d4372526"
   event_service_integration_tests:
     version: "2c51c024"
   application_gateway:
-    version: "PR-9406"
+    version: "2407bbc2"
   application_gateway_tests:
-    version: "PR-9406"
+    version: "2407bbc2"
   application_gateway_legacy_tests:
-    version: "PR-9394"
+    version: "4eeef0b7"
   application_registry:
-    version: "PR-9406"
+    version: "2407bbc2"
   application_registry_tests:
-    version: "PR-9356"
+    version: "91ae6ff1"
   application_broker:
```

Comment:
Can we use version 2f501fd2 here?

Reply:
Let's leave it as it is for now and bump it later.